conda_smithy/templates/upload_package.py.tmpl

```diff
@@ -0,0 +1,27 @@
+# This file was generated automatically from conda-smithy. To update this configuration,
+# update the conda-forge.yml and/or the recipe/meta.yaml.
+
+from yaml import safe_load
+import subprocess
+import json
+import sys
+
+global_config = json.loads('''
+{{ channels | tojson }}''')
```

Comment:
This is the jinja render that that we include here.  This provides the fallback for recipes that do not have the channel_targets / channel_sources in their conda_build_config files.

Reply:
Can we read .conda-forge.yml and get this value?

Reply:
We don't presently persist this anywhere.  I guess we could